### DISCUSSION TITLE
configuring the size of elements like the progress bar

### DISCUSSION THREAD

**Discussion by Ovens-Syntax** [TrevorSatori/kronos#20] [General]
hi, just reaching out to ask if there is a way too configure the size of some of the elements namely the progress bar

**Comment by TrevorSatori** [comment]
Hey, thanks for reaching out. After reading this I've updated the software to allow for progress bar size variability. To change the progress bar size, in the config.toml create a new header [layout] and underneath create a variable progress_bar = 10 or whatever number you want from 1 - 100.  I've updated the readme as well to demonstrate this. If there's any other suggestions keep 'em coming!

**Reply by Ovens-Syntax** [reply]
thanks! I forgot to put it in the og post but the program is excellent, Great work
